### ISSUE TITLE
ReStructuredText language support

### ISSUE THREAD

**Issue by adrian5** [arcticicestudio/nord-atom-syntax#37]
I love to work with this theme, but [ReStructuredText](https://github.com/Lukasa/language-restructuredtext) (a documentation language I use a lot) comes off very monotone, even for my tastes.

If more use could be made of nord-atom's color palette, I think it would improve the readability of ReST documents.

Thanks!

**Comment by arcticicestudio** [comment]
Thanks for your proposal 👍 
I rarely used ReST docs and thought the `markup` or `markdown`/`gfm` selectors would include th scope for these files, but it looks indeed a bit too monotone because some element selectors are language specific.

I've implemented optimized styles for
  - normal, phase and anonymous references
  - interpreted texts
  - fields lists
  - line-block symbols

You can test it by checking out the 🌱 [feature/ghi-#37-restructuredtext-language-support](https://github.com/arcticicestudio/nord-atom-syntax/tree/feature/ghi-%2337-restructuredtext-language-support) branch.

**Comment by adrian5** [comment]
Thanks for the quick update!

What do you think about having **bold** and _emphasized_ spans sharing the color of the asterisks surrounding them? I find it makes them stand out in a good way and could even see one of them taking on the pinkish hue from the nord palette (usually used by numbers I think).

If however, that would go against your minimalist design philosophy, I can understand that angle.

**Comment by arcticicestudio** [comment]
I like the idea to have **bold** and *italic* elements to be colored with their key characters, you can pull the latest commit to get the changes 😄 
But if you'd like to colorize them with `nord15` feel free to add it to your `~/.atom/styles.less` file:
```less
  .syntax--markup {
    &.syntax--bold,
    &.syntax--italic {
      color: @syntax-color-number;
    }
  }
```

**Comment by arcticicestudio** [comment]
![ghi-37-scrot-preview-1](https://cloud.githubusercontent.com/assets/7836623/24577665/8b803448-16d2-11e7-96a0-abdd207aa08c.png)

![ghi-37-scrot-preview-2](https://cloud.githubusercontent.com/assets/7836623/24577668/8e0bb71e-16d2-11e7-8a7d-4fd0c115f08c.png)

![ghi-37-scrot-preview-3](https://cloud.githubusercontent.com/assets/7836623/24577669/8fff0184-16d2-11e7-9198-118fb9e6febd.png)

**Comment by arcticicestudio** [comment]
Sorry for the delay 😉 
🚢 Shipped in APM 📦 package release version 🏷  [0.7.0](https://github.com/arcticicestudio/nord-atom-syntax/releases/tag/v0.7.0).

**Comment by adrian5** [comment]
Just received the update over the regular channel. Really digging that final color composition! Thanks for your efforts on this _literally_ cool theme, it helps me stay focused  :relieved:
